config/controller/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/kms-controller
-  newTag: v0.1.2
+  newTag: v0.0.0-non-release-version
```

Comment:
Can you please pull the latest tags and regenerate the controller? we should keep the `v0.1.2` version here.

Reply:
I have put as `v0.1.2` now